daemon/daemon_unix.go

```diff
@@ -653,6 +653,9 @@ func configureKernelSecuritySupport(config *Config, driverName string) error {
 }
 
 func (daemon *Daemon) initNetworkController(config *Config, activeSandboxes map[string]interface{}) (libnetwork.NetworkController, error) {
+	if err := processPoolsConfig(config); err != nil {
+		return nil, err
+	}
```

Comment:
I do not think we will allow the pools to be modified on reload.
Currently only two networking configs are allowed to be changed on cofig reload `--cluster-store` and `--cluster-advertise`.
Even if in future we decide to support the default pool re-config, I don't think we should remove the existing networks using the old pools. User is configuring a default, which takes place from this moment on, it is understandable existing networks will still be using the pool assigned to them at time of creation.

The reason why we have the default pool controlled in the ipamutils pkg itself, is because they need to be available to both libnetwork and swarm.

We could let libnetwork controller handle the the default pool config, then program the ipamutils pkg, but then how can we guarantee docker or swarm code does wait for the network controller to be up before invoking ipamutils methods ?